src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Properties {

```

Comment:
add constructor with two params and use super(color) inside. Do that everywhere.